src/main/java/com/example/IDriver.java

```diff
@@ -0,0 +1,9 @@
+public interface IDriver {
```

Comment:
where clear ?

Reply:
we didn't write command for it so didn't add it